### ISSUE TITLE
Are there plans to support Rails version 5.1 or higher?

### ISSUE THREAD

**Issue by chimame** [hashicorp/vault-rails#58]
### Problems and suggestions
As far as gemspec is concerned, currently supported version is more than 4.1 and less than 5.1.
However, Rails has already released 5.2, and development of 6.0 has already begun.

I'd like to use this gem with version 5.2, but I can not use it because it is unsupported.

### Countermeasure
If there is a schedule corresponding to version 5.1 or higher, I will indicate that fact and I would like to identify the tasks necessary for response.

**Comment by chrislewis60** [comment]
Interestingly, when I added
```
gem "vault-rails", "~> 0.1", require: false
```

and bundled, it installed version 0.3.2. If you look at the gem dependencies on that tag, it's just:

```
s.add_dependency "rails", ">= 4.0"
```

So that seems to work for me on Rails 5.2. Strangely, 0.4.0 tag has reverted to having an upper limit of rails 5.0 again.

**Comment by noma4i** [comment]
Any updates so far?

**Comment by pat** [comment]
FWIW I've just created pull requests for Rails 5.1 support (#66) and 5.2 support (#67) - though the latter drops support for Rails 4.1. I'm yet to play with these changes in any production environments, mind you, but the tests pass.

**Comment by therealrodk** [comment]
+1

**Comment by michaelsalexander83** [comment]
Any updates on this?
I would like to use this gem in an application that is rails 5.2.3

**Comment by mkralik** [comment]
rails 6.* is also impacted

**Comment by chrisarcand** [comment]
Hello everyone! Sorry for the delay and lack of info here.

There are indeed plans to update this gem with newer supported versions of Rails. We'll try and provide some more insight in to this topic when we can. 

Thanks so much for your patience!

**Comment by fatfrog** [comment]
I'm supposed to be asking my boss for $10K in a month to buy an enterprise license and the pace of development for this gem is not exactly helping my case.

**Comment by kalafut** [comment]
Fixed in #98
